src/net.h

```diff
@@ -719,6 +719,7 @@ class CNode
     std::atomic_bool fPauseRecv{false};
     std::atomic_bool fPauseSend{false};
 
+    bool m_addr_relay_peer;
```

Comment:
Now that this is no longer `const`, it should be initialized to something. Presumably `false`?
```suggestion
    bool m_addr_relay_peer{false};
```